eth/protocols/eth/protocol.go

```diff
@@ -39,11 +40,11 @@ const ProtocolName = "eth"
 
 // ProtocolVersions are the supported versions of the `eth` protocol (first
 // is primary).
-var ProtocolVersions = []uint{ETH68}
+var ProtocolVersions = []uint{ETH69, ETH68}
 
 // protocolLengths are the number of implemented message corresponding to
 // different protocol versions.
-var protocolLengths = map[uint]uint64{ETH68: 17}
+var protocolLengths = map[uint]uint64{ETH68: 17, ETH69: 17}
```

Comment:
I think this is wrong...

Reply:
It's a mapping. There are gaps in the protocol messages, but we need to know the highest numbers. It's a PITA multiplexer in the devp2p.